### ISSUE TITLE
mean curves out of confidence interval in plotDataCI

### ISSUE THREAD

**Issue by Quentin62** [modal-inria/MixtComp#2]
[res.zip](https://github.com/modal-inria/MixtComp/files/5218320/res.zip)

```
res = readRDS("res.rds")


plotDataCI(res, var = "func", class = 2)
```

![Rplot](https://user-images.githubusercontent.com/19777553/93059587-363e2080-f671-11ea-93e9-b6338c13f485.png)

**Comment by Quentin62** [comment]
the density is defined in the article (section 3.2 formula 7)
https://github.com/modal-inria/MixtComp/blob/master/MixtComp/docs/article/functionalModel.pdf


mean is defined by `sum_{l=1}^{nSub} kappa_{kl} \sum_{i=1}^{nCoeff} \beta_{kli}* t^{i-1}`

for upper and lower bounds, we define the function (say p) implementing formula 7

then, the goal is to find x such that p(x) is equal a requested probability (0.025 and 0.975 for computing a confidence interval).
See [functionalboundval](https://github.com/modal-inria/MixtComp/blob/master/RMixtCompUtilities/R/PLOT_extractCIbounds.R#L151) function 

At first glance, it seems correct

**Comment by Quentin62** [comment]
in fact, the `optimize` function does not found a very close solution. 

Instead:
Use the `uniroot` function with an other objective function

**Comment by Quentin62** [comment]
correction will be released in RMixtCompUtilities 4.1.3
